staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -70,8 +70,9 @@ func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 	azureSource := volume.AzureDisk
 	pv := &v1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
-			// A.K.A InnerVolumeSpecName required to match for Unmount
-			Name: volume.Name,
+			// Must be unique per disk as it is used as the unique part of the
+			// staging path
+			Name: azureSource.DataDiskURI,
```

Comment:
@davidz627 
pls use diskName, `DataDiskURI` is too long and it contains lots of `/` chars, code snippet is like following:
```
	diskName, err := getDiskName(diskURI)
	if err != nil {
		return nil, err
	}
```

Reply:
thanks for the review, fixed!